core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -41,20 +54,27 @@ contract FeePayer is Testable {
         collateralCurrency = IERC20(collateralAddress);
         finder = FinderInterface(finderAddress);
         lastPaymentTime = getCurrentTime();
+        cumulativeFeeMultiplier = FixedPoint.fromUnscaledUint(1);
     }
 
     /**
      * @notice Pays UMA DVM regular fees to the Store contract. These must be paid periodically for the life of the contract.
-     * @return the amount of collateral that was paid to the Store.
+     * @return the amount of collateral that the contract paid (sum of the amount paid to the store and the caller).
      */
-
     function payFees() public returns (FixedPoint.Unsigned memory totalPaid) {
```

Comment:
As discussed IRL, we might want to stop assessing fees after a `position` expires, after a `liquidation` expires, or after a `dispute` resolves. I'm thinking there are at least two ways we could do this:

- Pass in a "stopTime" parameter into `payFees` that checks if `(uint time = getCurrentTime()) > stopTime`. If so, then `time = stopTime`. 
- Have both `Liquidatable` and `PricelessPositionManager` overwrite the `payFees` method and insert its own `stopTime` logic

Thoughts? Maybe need to do in another PR

Reply:
Yeah, I think we'd want to change how liquidations work in tandem so we're consistent about how we tax collateral that's in the process of being settled. Right now, my understanding is that all collateral is taxed until withdrawn.